Kitodo/src/main/java/de/sub/goobi/forms/CalendarForm.java

```diff
@@ -642,10 +639,10 @@ public void unselectClick() {
 
     /**
      * The constant field TODAY hold the date of today. Reading the system clock
-     * requires much synchronisation throughout the JVM and is therefore only
-     * done once on form creation.
+     * requires synchronization throughout the JVM and is therefore only done
+     * once on form creation.
      */
-    private final LocalDate TODAY = LocalDate.now();
+    private final LocalDate kToday = LocalDate.now();
```

Comment:
If you want to change this variable name from `TODAY` to something more meaningful do not add a `k` letter in front of it. It is after this unreadable and not meaningful. Yes, I read your later explanation and I personally hate this and it is not covered by coding guideline nor check style rules.